modules/ROOT/pages/deploy-on-premises.adoc

```diff
@@ -57,10 +57,11 @@ mvn clean package deploy -DmuleDeploy
 |Parameter | Description
 |`standaloneDeployment` | Top-Level Element.
 include::partial$mmp-concept.adoc[tag=applicationNameParameterDescription]
-| `muleVersion` | The Mule version running in your local machine instance. +
-If this value does not match the Mule version running in your deployment target, the plugin raises an exception.
-
-The Mule Maven Plugin does not download a Mule runtime engine if these values don't match.
+| `muleVersion` | The release version of Mule runtime engine running in your local machine instance.  +
+Ensure that this value is equal to or higher than the earliest required Mule version of your application. +
+Example values: `4.2.0`, `4.2.2-hf4`. +
+If this value does not match the Mule version running in your deployment target, the plugin raises an exception. +
```

Comment:
It would be better to say "the Mule Maven Plugin in the first sentence and "The plugin..." in the second one - global